README.md

```diff
@@ -26,9 +26,9 @@ See [covid-data-public](https://github.com/covid-projections/covid-data-public)
 
 ## Running
 
-### Create JSON for UI
+### Run website data deploy
```

Comment:
Can you add a note explicitly saying that this will run all models and write them to appropriate locations in ../covid-projections/public/data/ ?